tests/view_tests/tests/test_debug.py

```diff
@@ -557,6 +563,25 @@ def __str__(self):
         text = reporter.get_traceback_text()
         self.assertIn('USER: [unable to retrieve the current user]', text)
 
+    def test_sensitive_settings_cleansed(self):
+        try:
+            request = self.rf.get('/test_view/', DOGECOIN_SETUP=self.DOGECOIN_SETUP)
+            request.user = User()
+            raise ValueError("Hello this is doge")
+        except ValueError:
+            exc_type, exc_value, tb = sys.exc_info()
+        reporter = ExceptionReporter(request, exc_type, exc_value, tb)
+        html = reporter.get_traceback_html()
+        pushing_asserts_below = []
+        out_of_trace_context = []
+        if not pushing_asserts_below and not out_of_trace_context:
+            self.assertNotIn('DOGECOIN_SETUP-nicekey', html)
```

Comment:
I'd use an approach from other tests that iterates over the key/values and checks the key is there but not the value.